### PR TITLE
[15.0][ADD] helpdesk_mgmt_ticket_required_fields

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="ticket_view_form" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.form (fields required)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='team_id']" position="attributes">
+                <attribute name="attrs">{'required': True}</attribute>
+            </xpath>
```
Comment (dalonsod) [inline helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml]: Locally tested :+1: 

Anyway, current code is potentially insecure, due to `attrs` overwriting (if other addon adds `attrs` for any field, our encoding or other addon's should be overwritten).
Could you try this instead, for every field, and test it again (we can use `required` attribute when a domain is not required)? If it works, add it to migration commit
```suggestion
            <xpath expr="//field[@name='team_id']" position="attributes">
                <attribute name="required">True</attribute>
            </xpath>
```

**Review by dalonsod — CHANGES_REQUESTED** [review]

File: helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml (hunk 1 above)
Comment (adriresu) [reply helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml]: Done, looks like It is not necessary to have a domain specified in order to use the required attribute.

Comment (dalonsod) [reply helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml]: OK, don't forget adding it (squashing with) migration commit (it's an improvement that we don't need to port to another version), and add yourself to contributors.

Comment (dalonsod) [reply helpdesk_mgmt_ticket_required_fields/views/helpdesk_ticket_views.xml]: Well, this is still pending... I'll do it, it should be merged right now

**Pull request merged**